lib/util/manifest_parser_utils.js

```diff
@@ -183,6 +184,9 @@ shaka.util.ManifestParserUtils.AUDIO_CODEC_REGEXPS_ = [
   /^flac$/,
   /^mp4a/,
   /^[ae]c-3$/,
+  /^ac-4$/,
+  /^dts[cx]$/,
+  /^[du]dts$/,
```

Comment:
Ok, all of the others I know about, or can find info about online, but what are `ddts` and `udts`?

Reply:
Found them here: https://chromium-review.googlesource.com/c/chromium/src/+/3352911/21/media/formats/mp4/fourccs.h

Reply:
But if you think it's not well-known, I'm willing to remove it.

Reply:
I don't think it needs to be removed, but perhaps a comment after the regex giving it a name would be helpful.

Reply:
I'm not 100% confident from my search results, but I'm guessing "dts" refers to "DTS Surround Sound" codecs: https://en.wikipedia.org/wiki/DTS_(company)#DTS_technologies

Reply:
Done!